### PR TITLE
Fixes Brig Door Timers

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Fixes Brig cell timers not allowing you to flash people*

*note: manually flashing prisoners by the removing of processing officers close was never prevented by this bug, though effectiveness may vary.

fixes https://github.com/ParadiseSS13/Paradise/issues/5139

:cl: Fox McCloud
fix: Fixes brig cell timers not having the option to flash people
/:cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
so it WASN'T A mapping error!

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -31,15 +31,15 @@
 	maptext_height = 26
 	maptext_width = 32
 
-/obj/machinery/door_timer/New()
+/obj/machinery/door_timer/initialize()
```
Comment (monster860) [inline code/game/machinery/doors/brigdoors.dm]: This line is the actual bugfix, right?

Comment (Fox-McCloud) [inline code/game/machinery/doors/brigdoors.dm]: Yuppers.

**Comment by counterfeitguise** [comment]
Gonna guess the rapidly flashing UI might be a server issue. At least this'll be fixed! Thanks Fox!

**Comment by Fox-McCloud** [comment]
@counterfeitguise

@monster860 actually fixed that in a PR that just got merged like......30 minutes ago.

**Pull request merged**